app/app/result/page.jsx

```diff
@@ -0,0 +1,102 @@
+'use client';
+import '../globals.css';
```

Comment:
Why import `globals.css`? They're already imported in the main layout, you don't need it here